superset-frontend/src/SqlLab/components/AceEditorWrapper.tsx

```diff
@@ -34,41 +33,44 @@ import {
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
-const propTypes = {
-  actions: PropTypes.object.isRequired,
-  onBlur: PropTypes.func,
-  sql: PropTypes.string.isRequired,
-  schemas: PropTypes.array,
-  tables: PropTypes.array,
-  functionNames: PropTypes.array,
-  extendedTables: PropTypes.array,
-  queryEditor: PropTypes.object.isRequired,
-  height: PropTypes.string,
-  hotkeys: PropTypes.arrayOf(
-    PropTypes.shape({
-      key: PropTypes.string.isRequired,
-      descr: PropTypes.string.isRequired,
-      func: PropTypes.func.isRequired,
-    }),
-  ),
-  onChange: PropTypes.func,
-};
+type HotKey = {
```

Comment:
typescript conversion of this file. I'm a little unfamiliar with some of the objects here, so there's more `any`s than I'd like

Reply:
every little bit helps!